test/routes/src/server/routes/t_launchpad.js

```diff
@@ -78,8 +78,12 @@ describe('The Launchpad API endpoint', () => {
           .reply(200, 'name: test-snap\n');
         const lp_api_url = conf.get('LP_API_URL');
         nock(lp_api_url)
-          .post('/devel/+snaps')
-          .query({ 'ws.op': 'new' })
+          .post('/devel/+snaps', {
+            'ws.op': 'new',
+            git_repository_url: 'https://github.com/anaccount/arepo',
+            processors: ['/+processors/amd64', '/+processors/armhf'],
```

Comment:
I was thinking that maybe it would be worth adding a test to check if expected default architectures are properly passed to LP.

But I guess this test would simply check if this nock mock expectations are fulfilled which in theory it already does - if different archs are passed nock shouldn't catch this request and tests should fail loudly...